test/proxy/transparent/TransparentUpgradeableProxy.behaviour.js

```diff
@@ -47,6 +47,25 @@ module.exports = function shouldBehaveLikeTransparentUpgradeableProxy(createProx
     });
   });
 
+  describe('proxyAdmin', function () {
```

Comment:
If I'm not mistaken we seem to be missing a test for the event emitted in the constructor.

Reply:
Correct. I added an extra test